src/Model/Serializer/ValueObject/Geography/TranslatedAddressDenormalizer.php

```diff
@@ -23,7 +23,7 @@ public function supportsDenormalization($data, $type, $format = null): bool
     /**
      * @inheritdoc
      */
```

Comment:
I think the `inheritdoc` can be removed?

Reply:
Not yet, because that gives issues for PHPStan because we need PHP Generics to make this work